titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/validator/JobSecurityValidatorConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ *  * Copyright 2019 Netflix, Inc.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.netflix.titus.runtime.endpoint.validator;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "titus.validate.job.security")
+public interface JobSecurityValidatorConfiguration {
+    @DefaultValue("true")
+    boolean iamValidatorEnabled();
```

Comment:
doesn't archaius expect JavaBeans conventions for accessors here? (i.e.: `isIamValidatorEnabled()`)

Reply:
Not sure, but happy to change it.